docs-gb/models/securing-endpoints.md

```diff
@@ -0,0 +1,105 @@
+# Securing model endpoints
+
+In enterprise use cases, you may need to control who can access the endpoints for deployed models or pipelines. While Seldon Core 2 doesn’t natively provide this functionality, it integrates with various service meshes that support these requirements. Though Seldon Core 2 is service-mesh agnostic, the example on this page demonstrates how to set up authentication and authorization to secure a model endpoint using the Istio service mesh.
```

Comment:
At the moment we're presenting this as a disadvantage "Seldon Core 2 doesn't natively provide this functionality". In fact, this is an advantage: end-users can re-use the normal auth mechanisms that they have in their cluster/environment (at the service mesh level, or what is provided at the cloud provider level -- apigee in GCP, Amazon API Gateway for AWS or something provider-agnostic like Gravitee). We don't need to add those examples in (as we don't then have ready-made configuration examples for them yet) but let's present this in a positive light.